crm_action/models/crm_action.py

```diff
@@ -0,0 +1,77 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    This module copyright (C) 2015 Savoir-faire Linux
+#    (<http://www.savoirfairelinux.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields, api
+
+
+class CrmAction(models.Model):
+    _name = 'crm.action'
+    _description = 'CRM Action'
+
+    lead_id = fields.Many2one(
+        'crm.lead', string='Lead', ondelete='cascade')
+
+    @api.onchange('lead_id')
+    def check_change(self):
+        lead = self.lead_id
+        if lead and lead.partner_id:
+            self.partner_id = lead.partner_id.id
+
+    partner_id = fields.Many2one(
+        'res.partner', string='Customer')
+
+    date = fields.Date(
+        'Date', required=True,
+        default=lambda self: fields.Date.context_today(self))
+
+    user_id = fields.Many2one(
+        'res.users', string='User', required=True,
+        default=lambda self: self.env.user)
+
+    def search_action_types(self):
+        return self.env['crm.action.type'].search(
+            [('is_active', '=', True)]).sorted(lambda a: a.priority)
```

Comment:
`operator.itemgetter('priority')` is considered more pythonic by some people.

But why don't you pass 'priority' as the order argument to search in this case?